madmom/audio/cepstrogram.py

```diff
@@ -197,7 +220,7 @@ def __new__(cls, spectrogram, filterbank=MelFilterbank,
             spectrogram = Spectrogram(spectrogram, **kwargs)
 
         # instantiate a Filterbank if needed
-        if issubclass(filterbank, Filterbank):
+        if inspect.isclass(filterbank) and issubclass(filterbank, Filterbank):
```

Comment:
Can you please give an example of when this check is needed? If it is a common case, we should consider the same logic at other places as well (e.g. `FilteredSpectrogram`).